net/stubby/Makefile

```diff
@@ -0,0 +1,59 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=stubby
+PKG_VERSION:=0.2.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=David Mora <iamperson347+public@gmail.com>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/getdnsapi/stubby/archive/
+PKG_HASH:=adf030a55426918933870f2d49a0caed93023bb1ec806efb255c3e7494985821
+
+PKG_FIXUP:=autoreconf
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/stubby/Default
+	TITLE:=stubby
+	URL:=https://dnsprivacy.org/wiki/display/DP/DNS+Privacy+Daemon+-+Stubby
+endef
+
+define Package/stubby/description
+	This package contains the Stubby daemon (which utilizes the getdns library).
+endef
+
+define Package/stubby
+	$(call Package/stubby/Default)
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=IP Addresses and Names
+	TITLE+= - (daemon that uses getdns)
+	USERID:=stubby=891:stubby=891
```

Comment:
what's the latest wisdom on USERID:=stubby=891:stubby=891 type things? did we ever 
               get to an autoassignment of them? or a registry?

Reply:
I definitely just picked the uid/gid randomly. The upstream source is using "stubby" as the username and group name.

 A registry sounds like a great idea to avoid conflicts.

Reply:
right, I was just checking elsewhere, this is as good as you can do right now I think.

Reply:
At what number does user start on OpenWrt? (on some linux first user is uid 500)